streams/test-utils/src/test/java/org/apache/kafka/streams/MockProcessorContextTest.java

```diff
@@ -186,38 +183,6 @@ public void process(final String key, final Long value) {
         assertFalse(context.committed());
     }
 
-    @Test
-    public void shouldStoreAndReturnStateStores() {
```

Comment:
Why do we remove this test? Should we rather rewrite it calling the new `init()` passing in a mocked `StateStoreContext` object?